assets/i18n/en.json

```diff
@@ -1,5 +1,6 @@
 {
   "I18N_ABOUT_PAGE_TITLE": "About us - Oppia",
+  "I18N_ACTION_APPLY_TO_TEACH_WITH_OPPIA": "Apply to Teach With Oppia",
```

Comment:
@anggorodewanto @mit0110 @szangaro @anafrancisca Please could I ask for some help in translating this phrase to Indonesian, Spanish and Portuguese? Note that "Teach with Oppia" is the name of a new program that we're starting -- you can learn more by reading [this](https://github.com/oppia/oppia/blob/19cce351cf147076db102de39daa11653e7e0501/core/templates/dev/head/pages/teach.html#L55). Thanks!